scan/src/subscriptions/ValidatorSub.re

```diff
@@ -70,6 +76,8 @@ let toExternal =
   details,
   tokens,
   commission: commissionRate *. 100.,
+  commissionMaxChange,
+  commissionMaxRate,
```

Comment:
Should they multiple by 100 here?

Reply:
I multiplied it by 100 when using it in ValidatorIndexPage before. But I've moved it to the sub to match the commission rate now.